src/folio_migration_tools/migration_tasks/loans_migrator.py

```diff
@@ -50,6 +52,23 @@ def __init__(
         task_configuration: TaskConfiguration,
         library_config: LibraryConfiguration,
     ):
+        try:
+            logging.info(
+                "Attempting to retrieve tenant timezone configuration..."
+            )
+            self.tenant_timezone_str = json.loads(self.folio_client.folio_get_single_object(
+                "/configurations/entries?query=(module==ORG%20and%20configName==localeSettings)"
+                )["configs"][0]["value"])["timezone"]
+            logging.info(
+                "Tenant timezone is: %s", 
+                self.tenant_timezone_str
+            )
+        except Exception:
+            logging.info(
+                'Tenant locale settings not available. Using "UTC".'
```

Comment:
I wonder if this information (and the information of the tenant timezone) should be added to the migration report. 
But no need now.